ArmoniK.Extension.CSharp.DllCommon/TaskLibraryDefinition.cs

```diff
@@ -86,18 +86,22 @@ public static TaskConfiguration AddTaskLibraryDefinition(this TaskConfiguration
   /// <param name="taskOptions">The task options to retrieve from.</param>
   /// <param name="libraryName">The name of the library.</param>
   /// <returns>The TaskLibraryDefinition associated with the specified library name.</returns>
-  public static TaskLibraryDefinition GetTaskLibraryDefinition(this TaskOptions taskOptions,
-                                                               string           libraryName)
+  public static TaskLibraryDefinition? GetTaskLibraryDefinition(this TaskOptions taskOptions,
```

Comment:
If you change TaskOptions.GetDynamicLibrary to throw in case of error, you would not need to make the output nullable.